### ISSUE TITLE
Cannot change month & year from Date Edit- Nullable Date? when use inside Blazor Data Grid

### ISSUE THREAD

**Issue by mdosmaneng** [DevExpress/Blazor#79]
Hi,
I use Date Edit-Nullable Date? Picker in Blazor Data Grid. I cannot change any month & year from the datepicker. If i use the same datepicker outside of the grid, i can change date, month & year easily but inside the grid i can't. The code for Date Edit-
`<DxDateEdit Date="@DischargeDateExp" CssClass="my-date-edit" DateChanged="@(e => UpdateData(ClientVisitGuid, CurrentLocationGUID, NameOfDischargeDateExp, e))" ClearButtonDisplayMode="DataEditorClearButtonDisplayMode.Auto"></DxDateEdit>`

**Comment by zabelin-vladimir** [comment]
Hello @mdosmaneng

Could you please clarify what you mean by using our DxDateEdit component in our DxDataGrid?
As far as I understand, you use it in some template. I tested the provided code in the *EditTemplate* template of the *DateEdit?* column and it works properly. Please review the project and try to modify it to reproduce this issue.
I look forward to your reply.

[GHI79.zip](https://github.com/DevExpress/Blazor/files/3982612/GHI79.zip)

**Comment by mdosmaneng** [comment]
Hi @zabelin-vladimir,
Thanks for your replay.

Actually, Our user not interested to update date with EditTemplate. They want to update only date  in a row in-cell specific rather than whole row in EditTemplate. 

**Note: The functionalities of DxDateEdit does not work properly.**

Please! find the sample project below...


[GHI79.zip](https://github.com/DevExpress/Blazor/files/3989052/GHI79.zip)

**Comment by zabelin-vladimir** [comment]
Hello @mdosmaneng,

Thank you for your sample project! I reproduced this issue on my side.
We will research this behavior and ways to resolve it in the context of the [DateEdit for Blazor placed inside DisplayTemplate of DxDataGrid - It's not possible to change the current month/year with the navigation buttons](https://www.devexpress.com/Support/Center/Question/Details/T847761/dateedit-for-blazor-placed-inside-displaytemplate-of-dxdatagrid-it-s-not-possible-to) Support Center ticket.
As a possible workaround, you can wrap DxDateEdit in some HTML element and stop event propagation for this element:
`<DisplayTemplate> <div @onclick:stopPropagation="true"> <DxDateEdit Date=@((context as WeatherForecast).Date) DateChanged="@(e => UpdateData(e))"></DxDateEdit> </div> </DisplayTemplate>`

**Comment by zabelin-vladimir** [comment]
Hello @mdosmaneng

We have fixed the issue from the [DateEdit for Blazor placed inside DisplayTemplate of DxDataGrid - It's not possible to change the current month/year with the navigation buttons](https://supportcenter.devexpress.com/ticket/details/t847761/dateedit-for-blazor-placed-inside-displaytemplate-of-dxdatagrid-it-s-not-possible-to) SC ticket in [our 19.2.1-beta release](https://github.com/DevExpress/Blazor/releases/tag/v19.2.1-beta).
I checked your project with this version and it works properly. This release is available via the DevExpress NuGet server. Refer to [this page](https://nuget.devexpress.com/) to obtain your personal NuGet feed URL.
